README.md

```diff
@@ -128,6 +128,14 @@ jobs:
         port: 8080
 ```
 
+## <a id="multi-arch-builds">Multi arch builds</a>
+
+For building image(s) with multiple architecture(s) suggest running,
```

Comment:
if building for an architecture other than `amd64`, install `qemu-user-static` using the following command.